src/main/java/mate/academy/internetshop/dao/jdbc/JdbcProductDao.java

```diff
@@ -72,17 +68,12 @@ public Product create(Product product) {
     @Override
     public Product update(Product product) {
         try (Connection con = ConnectionUtil.getConnection()) {
-            String query = "UPDATE products SET name = ?, price = ? WHERE id = ?";
+            String query = "UPDATE products SET name = ?, price = ? WHERE product_id = ?";
             PreparedStatement statement = con.prepareStatement(query);
             statement.setString(1, product.getName());
             statement.setBigDecimal(2, product.getPrice());
             statement.setLong(3, product.getId());
-            if (statement.executeUpdate() > 0) {
-                return get(product.getId()).get();
-            }
-            String msg = "Enable to update product with id " + product.getId();
-            LOGGER.error(msg);
-            throw new RuntimeException(msg);
+            return get(product.getId()).get();
```

Comment:
you already have product  that passed into your method with a new values inside

Reply:
price field may change because of type convercion after saving to database

Reply:
what type you are using for price column?

Reply:
BigDecimal in java and Decimal(10, 0) in SQL

Reply:
I've deleted that code, but I think it has some sense, but we in any case dosen't use value that returned by update method

Reply:
try to use Decimal(10,2) , that should reduce the difference